TrafficReplayer/src/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -245,6 +323,44 @@ public void consumeLogLinesForFile(Path p, Consumer<ReplayEntry> replayEntryCons
         }
     }
 
+    private static String formatPacketsAsBase64String(String prefix, ArrayList<byte[]> packets) {
```

Comment:
Is this used?  I was wondering what the prefix was, but I don't see a reference to it in the PR & the code shows as new.

Reply:
Nope, not used. Thanks for catching. It's new in the scope of this PR because it went with the original approach of outputting everything as base64 strings, but not used in the latest commits.